src/components/Admin/Admin.js

```diff
@@ -76,7 +76,7 @@ export const Admin = createReactClass({
                     <Tab>Geofence / Monitored places</Tab>
                     <Tab>Event Import</Tab>
                     <Tab>Facebook pages</Tab>
```

Comment:
we dont need facebook pages as trusted sources should support that going forward.

Reply:
ok